modules/nixos/main.py

```diff
@@ -260,7 +261,13 @@
 """
 
 cfgfirefox = """  # Install firefox.
-  programs.firefox.enable = true;
+  programs.firefox = {
+    enable = true;
+    package = pkgs.firefox;
+    nativeMessagingHosts.packages = with pkgs; [
+      browserpass
```

Comment:
If installing it is a too big change, we could do the same than for Thunderbird, just suggesting it by writting it but commenting it like
```
    nativeMessagingHosts.packages = with pkgs; [
      # browserpass
```
This way, it won't change installation process at all but it will make it very much easier for new comers to find out how to activate it and make Firefox work with Gnome extension and also.

It would also give indication where the user should add thing like
`gnomeExtensions.gsconnect` or `firefoxpwa` : 

```
    nativeMessagingHosts.packages = with pkgs; [
      browserpass
      gnomeExtensions.gsconnect
      firefoxpwa
    ];
  };
```